docs/how_to/env_var.md

```diff
@@ -3,36 +3,52 @@ Environment Variables
 MXNet has several settings that you can change with environment variables.
 Typically, you wouldn't need to change these settings, but they are listed here for reference.
 
+For example, you can set these environment variables in linux or mac as follows:
+```
+export MXNET_GPU_WORKER_NTHREADS=3
+```
+
 ## Set the Number of Threads
 
-* MXNET_GPU_WORKER_NTHREADS (default=2)
-  - The maximum number of threads that do the computation job on each GPU.
-* MXNET_GPU_COPY_NTHREADS (default=1)
-  - The maximum number of threads that do the memory copy job on each GPU.
-* MXNET_CPU_WORKER_NTHREADS (default=1)
-  - The maximum number of threads that do the CPU computation job.
-* MXNET_CPU_PRIORITY_NTHREADS (default=4)
- - The number of threads given to prioritized CPU jobs.
-* MXNET_CPU_NNPACK_NTHREADS (default=4)
- - The number of threads used for NNPACK.
+* MXNET_GPU_WORKER_NTHREADS
+  - Values: Int ```(default=2)```
+  - The maximum number of streams to use on each GPU. This parameter is used to parallelize the computation within a single GPU card.
+* MXNET_GPU_COPY_NTHREADS
+  - Values: Int ```(default=1)```
+  - The maximum number of concurrent threads that do the memory copy job on each GPU.
+* MXNET_CPU_WORKER_NTHREADS
```

Comment:
Curious: What happens if we specify the number of threads to be more than the capacity of the GPU? We may not need to document but a user may have a similar question. BTW some enterprise products document the range of permissible values to help answer that question. See here: 
https://www.ibm.com/support/knowledgecenter/en/SSGU8G_12.1.0/com.ibm.sqlr.doc/ids_sqr_401.htm